indico/modules/events/papers/controllers/display.py

```diff
@@ -159,3 +162,20 @@ class RHDeletePaperComment(RHPaperCommentBase):
     def _process(self):
         delete_comment(self.comment)
         return jsonify_data(flash=False)
+
+
+class RHDisplayReviewingArea(RHPapersBase):
+    def _checkProtection(self):
+        if not session.user:
+            raise Forbidden
+        if session.user not in self.event_new.cfp.layout_reviewers and \
```

Comment:
Instead of getting the list of reviewers, check whether the user has the ACL role

    event.can_manage(user, role='XXX', explicit_role=True)

XXX would be 'paper_content_reviewer' or 'paper_layout_reviewer'